content/en/docs/contribute/review/reviewing.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Reviewing pull requests
+content_template: templates/concept
+main_menu: true
+weight: 10
+---
+
+{{% capture overview %}}
+
+Anyone can review a documentation pull request. Visit the [pull requests](https://github.com/kubernetes/website/pulls) section in the Kubernetes website repository to see open pull requests.
+
+Reviewing documentation pull requests is a
+great way to introduce yourself to contributing to the Kubernetes community.
+It helps you learn the code base and build trust with other contributors.
+
+Before reviewing, it's a good idea to:
+
+- Read the  [content guide](/docs/contribute/style/content-guide/) and
+[style guide](/docs/contribute/style/style-guide/) so you can leave informed comments.
+- Understand the different [roles and responsibilities](/docs/contribute/participating/#roles-and-responsibilities) in the Kubernetes documentation community
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Before you begin
+
+Before you start a review:
+
+- Read the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) and ensure that you abide by it at all times.
+- Be polite, considerate, and helpful.
+- Comment on positive aspects of PRs as well.
+- Be empathetic and mindful of how your review may be received.
+- Assume good intent and ask clarifying questions.
+- Experienced contributors, consider pairing with new contributors whose work requires extensive changes.
+
+## Review process
+
+In general, review pull requests for content and style in English.
+
+1.  Go to
+    [https://github.com/kubernetes/website/pulls](https://github.com/kubernetes/website/pulls).
+    You see a list of every open pull request against the Kubernetes website and
+    docs.
+
+2.  Apply some or all of the following labels:
+    - `cncf-cla: yes` (Recommended): PRs submitted by contributors who have not signed the CLA cannot be merged. See [Sign the CLA](/docs/contribute/new-content/#sign-the-cla) for more information.
+    - `language/en` (Recommended): Filters for english language PRs only.
+    - `size/<size>`: filters for PRs of a certain size. If you're new, start with smaller PRs.
+
+    Additionally, ensure the PR isn't marked as a work in progress.
```

Comment:
Do you define what a WIP (work in progress) is and how to make your pull request a WIP?